js/models/messages.js

```diff
@@ -1087,6 +1087,9 @@
         }
 
         const { body, attachments, preview, quote } = await this.uploadData();
+        const ourNumber = window.storage.get('primaryDevicePubKey');
+        const ourConversation = window.ConversationController.get(ourNumber);
+        const lokiProfile = ourConversation.getOurProfile();
```

Comment:
`ourConversation` could be undefined for some reason

Reply:
updated